src/frontend/komponenter/Behandling/Personopplysninger/Personopplysninger.tsx

```diff
@@ -9,9 +9,10 @@ import Statsborgerskap from './Statsborgerskap';
 import DataViewer from '../../Felleskomponenter/DataViewer/DataViewer';
 import Oppholdstillatelse from './Oppholdstillatelse';
 import { useBehandling } from '../../../context/BehandlingContext';
+import NavKontor from './NavKontor';
 
 const Personopplysninger: React.FC = () => {
-    const { personopplysningerResponse } = useBehandling();
+    const { personopplysningerResponse, navKontorResponse } = useBehandling();
 
     return (
         <DataViewer response={{ personopplysningerResponse }}>
```

Comment:
Du kan tekntisk sett legge både navkontorResponse og personopplysningerResponse i samme DataViewer-response - da slipper du unna to "DataViewer" i én bolk

Reply:
Men hvis en av dem feiler vil ikke den andre vises tror jeg? norg2 kallet mot dev er fortsatt litt ustabilt, så ungår gjerne at det "blokkerer" for resten av personopplysninger.

Reply:
Et godt poeng, det :)